treeherder/seta/tests.py

```diff
@@ -0,0 +1,30 @@
+import datetime
```

Comment:
You should create these tests under `tests/` from the root directory in treeherder (probably create a subdirectory for seta: `tests/seta/*`, and follow the patterns we use elsewhere (i.e. using py.test instead of Django).